executor/src/main/scala/org/apache/amaterasu/executor/common/executors/ActiveNotifier.scala

```diff
@@ -0,0 +1,72 @@
+package org.apache.amaterasu.executor.common.executors
+
+import javax.jms.{DeliveryMode, MessageProducer, Session}
+
+
+import net.liftweb.json._
+import net.liftweb.json.Serialization.write
+import org.apache.activemq.ActiveMQConnectionFactory
+import org.apache.amaterasu.common.execution.actions.{Notification, NotificationLevel, NotificationType, Notifier}
+import org.apache.amaterasu.common.logging.Logging
+
+class ActiveNotifier extends Notifier with Logging {
```

Comment:
I think we should add to the notifier interface, a source string, to represent the message creator (container id / executor id or some other unique identifier)

Reply:
Seems to belong to the Notification class rather the Notifier interface.
Anyway I suggest this will be added as a new improvement card rather than change to AMATERASU-15